docsConfig

```diff
@@ -784,7 +784,7 @@ EXCLUDE_SYMLINKS       = NO
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories for example use the pattern */test/*
 
-EXCLUDE_PATTERNS       = */util/* */iTest/* */test/* */collection/* */connection/* */executor/* */impl/* */map/* */multimap/* */protocol/* */proxy/* */queue/* */serialization/pimpl/* */spi/* */topic/* */txn/* */exception/pimpl/*
+EXCLUDE_PATTERNS       = */util/* */iTest/* */test/* */collection/* */connection/* */executor/* */impl/* */map/* */multimap/* */protocol/* */proxy/* */queue/* */serialization/pimpl/* */spi/* */txn/* */exception/pimpl/*
```

Comment:
i think this whole pattern needs to be updated. it looks out of date. we can do it in another PR. Can you open an issue to update this file?

Reply:
Sure, I think we may also consider to review all documentation comments, doxygen gives lots of warnings. There are Javadoc things in the docs that are not supported or should be done differently in doxygen.

Reply:
Resetted this change, opened a new issue: https://github.com/hazelcast/hazelcast-cpp-client/issues/609.

Another question, is there a specific reason why the the config file is named docsConfig? Can we rename it to Doxyfile? It is the standard thing to do and one of the things that someone will look for when they first see the project.

Reply:
no reason but will need to update the release scripts when it is changed.

Reply:
Ok, I am noting this in #609